pennylane/ops/qubit.py

```diff
@@ -867,10 +884,12 @@ def __init__(self, *params, wires=None, do_queue=True):
                 " Allowed characters are I, X, Y and Z".format(pauli_word)
             )
 
-        if not len(pauli_word) == len(wires):
+        num_wires = 1 if isinstance(wires, int) else len(wires)
+
+        if not len(pauli_word) == num_wires:
```

Comment:
:100:

Reply:
Maybe a small test case catching this could be worth adding

Reply:
Good call, writing it made me realize this was not handled properly in `decomposition` either!